### PR TITLE
Added seperate buttons for Start / Stop on Recording and Streaming

### PR DESCRIPTION
**Pull request by mattehalen**
I misted the ability to start and stop recordings without the toggle button so this ads those function to the OBS module.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -82,12 +82,10 @@ instance.prototype.init = function() {
 		});
 
 		self.obs.on('ConnectionClosed', function() {
-			if (self.disable != true) {
```
Comment (eric) [inline index.js:85]: What is this change for?

File: index.js [hunk 2]
```diff
@@ -150,9 +148,7 @@ instance.prototype.init = function() {
 				self.states['studio_mode'] = false;
 			}
 		});
-		
 		self.obs.on('SceneItemVisibilityChanged', function(data) {
-			if (self.states['studio_mode'] == true) {
```
Comment (eric) [inline index.js:155]: Was this removal intentional?

**Review by eric — COMMENTED** [review]
It looks like there were some unrelated changes included in this.

File: index.js (hunk 2 above)
Comment (mattehalen) [reply index.js:155]: The removals is not intentional but from what I can see from my side it seams to be tabs thats removed when saving this file with my VSCode.

image

Comment (eric) [reply index.js:155]: Something strange is going on... on GitHub it looks like the entire lines were removed. Do you have another commit you needed to push?

File: index.js [hunk 3]
```diff
@@ -359,6 +355,18 @@ instance.prototype.actions = function() {
...
+		},
+		'StopRecording': {
+			label: 'Stop Recording',
+		},
+		'StartStreaming': {
+			label: 'Start Streaming',
+		},
+		'StopStreaming': {
+			label: 'Stop Streaming',
+		},
```
Comment (eric) [inline index.js:369]: To make these consistent with the rest of the code base, these actions should be in the format of `start_recording` instead of `StartRecording`

File: index.js (hunk 2 above)
Comment (mattehalen) [reply index.js:155]: i can try making a commit from Atom instead of VScode to see if that works better.

File: index.js (hunk 3 above)
Comment (mattehalen) [reply index.js:369]: I will change this in the next commit.

Comment (mattehalen) [reply index.js:369]: Made a new pull request with only the right changes but something strange is happening when i save this project. Those lines at 84 and 155 gets removed even if I dont touch them.

**Comment by eric** [comment]
You can close this one. The other one looks good.
